.github/workflows/test-opencast-site.yml

```
@@ -67,17 +67,27 @@ jobs:
         run: |
           .github/check-jacoco-deps.sh
 
+      - name: build opencast, skipping tests which run during site
+        run: |
+          ./mvnw install -DskipTests -T1C -Pnone \
+            --batch-mode \
+            -Dsurefire.rerunFailingTestsCount=2 \
```

We don't actually need this since we don't run any tests, do we?
We could skip the style checker, though.
```suggestion
            -Dcheckstyle.skip=true \
```

---

Yep, great catch